### PR TITLE
Adding relevant changes in playbooks to support ACI CNI for OSP-4.6:

### PR DESCRIPTION
**Pull request by abhis2112**
1. Changed Ignition Version from 2.2.0 to 3.1.0 in template/files/update_ign.py scripts
2. Moved addition of dns server to the network from aci-containers-network to primary network
3. Adjusted common changes to playbooks according to upstream UPI-4.6 respository
4. Removed aci-containers-port creation from bootstrap machine (Not needed) via changes in update_ign.py script
5. Added control-unsched.py script to unschedule master before install

### CODE REVIEW

File: upi/openstack/network.yaml [hunk 1]
```diff
@@ -38,10 +38,11 @@
...
 
-  - name: 'Create the cluster address-scope'
-    command:
-      cmd: "neutron address-scope-create node_network_address_scope 4 --apic:distinguished_names type=dict VRF={{ aci_cni['network_interfaces']['node']['vrf'] }}"
-    when: os_networking_type == "CiscoACI"
+    #  - name: 'Create the cluster address-scope'
+    #    command:
+    #      cmd: "neutron address-scope-create node_network_address_scope 4 --apic:distinguished_names type=dict VRF={{ aci_cni['network_interfaces']['node']['vrf'] }}"
+    #    when: os_networking_type == "CiscoACI"
+    #    ignore_errors: True
```
Comment (snaiksat) [inline upi/openstack/network.yaml:45]: I think you disabled this for testing, but should be uncommented. ;-)

**Comment by snaiksat** [comment]
What about reverting the changes to disabling the validation check?

File: upi/openstack/network.yaml (hunk 1 above)
Comment (abhis2112) [reply upi/openstack/network.yaml:45]: ahh right, nice catch, thanks, resolving it

**Comment by abhis2112** [comment]
> What about reverting the changes to disabling the validation check?

We can do that, can remove that

**Review by apoorva11029 — APPROVED** [review]

**Pull request merged**